### PR TITLE
added an option for changing the thumbnail size from normal to large in gvfamilylines.py

### PR DESCRIPTION
**Pull request by MJBMZ**
hi,

i added an option for changing the thumbnail size from normal to large in gvfamilylines.py.

github and coding is new for me, so i hope this is the right way to contribute the change.

matthias

### CODE REVIEW

**Comment by Nick-Hall** [comment]
Almost.  You should have created the pull request directly from your MJBMZ-patch-1 branch.  This would have avoided the extra commit that we don't need.

File: gramps/plugins/graph/gvfamilylines.py [hunk 1]
```diff
@@ -72,6 +72,9 @@
            {'name' : _("Colored outline"), 'value' : "colored"},
            {'name' : _("Color fill"), 'value' : "filled"}]
 
+SIZE_NORMAL = 0
```
Comment (Nick-Hall) [inline gramps/plugins/graph/gvfamilylines.py]: Import these constants from `gramps.gen.utils.thumbnails`

File: gramps/plugins/graph/gvfamilylines.py [hunk 2]
```diff
@@ -236,6 +239,12 @@ def add_menu_options(self, menu):
                                        'should appear relative to the name'))
         add_option('imageonside', self.image_location)
 
+        self.image_size = EnumeratedListOption(_('Thumbnail size'), 0)
+        self.image_size.add_item(0, _('Normal'))
```
Comment (Nick-Hall) [inline gramps/plugins/graph/gvfamilylines.py]: Use the constants here instead of 0 and 1.

File: gramps/plugins/graph/gvfamilylines.py [hunk 3]
```diff
@@ -809,9 +819,16 @@ def write_people(self):
                     media = self._db.get_media_from_handle(media_handle)
                     media_mime_type = media.get_mime_type()
                     if media_mime_type[0:5] == "image":
-                        image_path = get_thumbnail_path(
-                            media_path_full(self._db, media.get_path()),
-                            rectangle=media_list[0].get_rectangle())
+                        if self._imagesize == SIZE_NORMAL:
```
Comment (Nick-Hall) [inline gramps/plugins/graph/gvfamilylines.py]: The `if` statement is not necessary here.  Use `size=self._imagesize` in the call to `get_thumbnail_path`.

**Comment by MJBMZ** [comment]
Thank you Nick, i changed the parts.

**Comment by Nick-Hall** [comment]
In testing, I noticed that this option should be made inactive when thumbnails are not selected.  The thumbnail location option already does this, so it should be easy to copy the code.

**Comment by Paul-Franklin** [comment]
As long as you still have changes to make, you might as
well change "EnumeratedListOption(_('Thumbnail size'), 0)"
so that the zero becomes a SIZE_NORMAL too.

**Comment by MJBMZ** [comment]
I have read about merging or squashing these different commits, but i'm not sure if this is something i can do or where. can i do something to clean this up?

**Comment by prculley** [comment]
read up on "git rebase -i yourcommithash".  If you have your text editor wet up to start when git needs it, then it will pop up a text file with the commits after the one you specify.  You can then use a 'squash' or 'fixup' to merge several commits into a single one and you can update the commit message as well.

**Comment by MJBMZ** [comment]
thanks prculley, worked fine (after installing git and learning how to use it, this was the first file i ever edited so i used the github webui before...)

**Comment by Nick-Hall** [comment]
Thanks.  I just made one minor change.  Merged a8a3c0865d52390c08dc518b98922f1f1a86a4c4.

**Pull request closed without merging**